accounting/on_chain.go

```diff
@@ -178,9 +178,8 @@ func onChainReportWithPrices(cfg *OnChainConfig, getPrice usdPrice) (Report,
 		return nil, err
 	}
 
-	isSweep := make(map[string]bool, len(sweeps))
 	for _, sweep := range sweeps {
-		isSweep[sweep] = true
+		info.sweeps[sweep] = true
```

Comment:
Commit 77b451b759a61fde7aff00f3aac0d2a108132b17 has the itest binaries committed, they should be removed and/or added to gitignore.